### PR TITLE
v0.9.0.1

### PR DESCRIPTION
**Pull request by jjr840430**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
-# libmulticonfig
\ No newline at end of file
+
+# Multi-configuration parser library using C
+
+`libmulticonfig` is a multi-configuration parser library which was implemented by C.
```
Comment (erikbor) [inline README.md]: s/parser library which was implemented by C/parser library implemented in C/

File: README.md [hunk 2]
```diff
@@ -1 +1,78 @@
...
+The several configurations may be parsed by this library:
```
Comment (erikbor) [inline README.md]: s/The several configurations may be parsed by this library:/The configuration for the library is parsed in this order:/

**Comment by jjr840430** [comment]
@ronaldtse @erikbor I uploaded the first version of `README`. It may be poor but let me know if you have any updates. Thanks!

**Comment by jjr840430** [comment]
@ronaldtse @erikbor I will merge it right now. Will solve issues in next PR.

**Pull request merged**